Makefile.am

```diff
@@ -232,7 +248,22 @@ cpp_server_ct_server_SOURCES = \
 	cpp/server/certificate_handler.cc \
 	cpp/server/handler.cc \
 	cpp/server/json_output.cc \
-        cpp/server/log_processes.cc \
+  cpp/server/log_processes.cc \
```

Comment:
Wasn't tabs, for some reason it's not displaying correctly in my editor. It worked in vi so I'm about to push a fix.